.gitignore

```diff
@@ -411,6 +411,7 @@ modules.xml
 # Project specific ignores
 /build.properties
 /data/
+**/respondr/defaultSkin.css
```

Comment:
I'm compiling the file into the source folder to avoid modifying the `war` step.